docs/install/usb-install.md

```diff
@@ -28,6 +28,12 @@ Make sure you choose the correct device. The process erases data on the selected
 # sudo dd if=<path_to_iso> of=<path_to_usb_device> bs=64k
 ```
 
+Sometimes using a USB to mount the Harvester ISO disk directs you to Grub2. In this case, it helps to know what to set the root partition to and the location of `grub.cfg`.
+
+You can see the current root partition in the `grub.cfg`. To view the `grub.cfg` from the ISO, run the command `chainloader /boot/grub2/x86_64-efi/grub.efi`.
+
+To set a new root partition, run the command `set root = (hdN,msdos1)`, where N denotes N numbers of possible drives. N should be the drive your USB is mounted to.
```

Comment:
the command should read:
`set root=(hdN,msdos1)` 

as if it has " " (spaces) in the command, the command will fail if I remember correctly :thinking: :smile: